checkov/common/typing.py

```diff
@@ -13,6 +13,10 @@
     [str, "BaseCheck", "_SkippedCheck", "dict[str, Any]", str, str, "dict[str, Any]"], None
 ]
 
+_Resource = str
+_Attribute = str
+ResourceAttributesToOmit = Dict[_Resource, _Attribute]
```

Comment:
```suggestion
ResourceAttributesToOmit: TypeAlias = dict[str, str]
```
something like this is called a `TypeAlias` and should be marked as such 🙂 
I don't think you need `_Resource` and `_Attribute`, at least I didn't see you using it somewhere.

Reply:
I dont use them, my intention was to indicate the expected structure of `ResourceAttributesToOmit` if anyone wants to use it in the future, so its clear that the key should be a resource name and the value its matching attribute name.
Think its better not to mention that? @gruebel

Reply:
gotcha, sounds good, then
```python
_Resource: TypeAlias = str
_Attribute: TypeAlias = str
ResourceAttributesToOmit: TypeAlias = dict[_Resource, _Attribute]
```